geospaas_processing/copiers.py

```diff
@@ -76,3 +78,18 @@ def copy(self):
             else:
                 LOGGER.debug("For dataset with id = %s, there is no local file address in the "
                              "database.", dataset.id)
+
+    def delete(self):
+        """
+        Delete the file(s) or symlink(s) after a certain period of obsoleteness (in days) of the
+        file(s) or symlink(s) inside the destination path. """
+        with os.scandir(self._destination_path) as scanned_dir:
+            for entry in scanned_dir:
+                if ('.snapshot' not in entry.path
+                and entry.stat(follow_symlinks=False).st_uid == os.getuid()
+                and time.time() - entry.stat(follow_symlinks=False).st_mtime >
+                    self.obsoleteness*24*3600):
```

Comment:
Please fix the indentation.

Reply:
How? Do you mean removing it? I dont know that you mean exactly.

Reply:
Add 4 or 8 spaces in the beginning of the lines which begin with `and`.

Reply:
Done in the next commit